hollow/src/main/java/com/netflix/hollow/api/codegen/HollowConsumerJavaFileGenerator.java

```diff
@@ -118,7 +118,7 @@ protected void appendPackageAndCommonImports(StringBuilder builder,
                     }
                 }
                 for (String schemaName : schemaNameSet) {
-                    appendImportFromBasePackage(builder, schemaName + config.getClassPostfix());
```

Comment:
for the actual generated class, is it always uppercase?

Reply:
Yes we do that here https://github.com/Netflix/hollow/blob/45e19a67d3db4432085874af03809c3cd5c88e7e/hollow/src/main/java/com/netflix/hollow/api/codegen/HollowCodeGenerationUtils.java#L164,L177

Accessors seem the only place where we don't